### PR TITLE
fix: Documentation link

### PR DESCRIPTION
**Pull request by embano1**
Issue #, if available:

Description of changes:

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

**Comment by RedbackThomson** [comment]
I'll wait until those 2 members join the appropriate Github teams before we merge this
/hold

File: metadata.yaml [hunk 1]
```diff
@@ -1,8 +1,8 @@
 service:
   full_name: "Amazon EventBridge"
   short_name: "EventBridge"
-  link: https://aws.amazon.com/eventbridge/"
-  documentation: https://docs.aws.amazon.com/eventbridge/latest/APIReference/Welcome.html"
+  link: "https://aws.amazon.com/eventbridge/"
+  documentation: "https://docs.aws.amazon.com/eventbridge/latest/APIReference/Welcome.html"
```
Comment (baldawar) [inline metadata.yaml:5]: Does this organically change for other partitions? e.g. https://docs.amazonaws.cn/en_us/eventbridge/latest/APIReference/Welcome.html

Comment (embano1) [reply metadata.yaml:5]: Unlikely, this logic lives in `OLM` (Operator Hub) and we're just providing the bare minimal information here. Considering this non-blocking.

**Review by A-Hilaly — APPROVED** [review]
:+1: 
/lgtm

**Pull request merged**